### ISSUE TITLE
Menu - accessibility - on desktop the mobile burger icon should have aria-hidden=true 

### ISSUE THREAD

**Issue by Dan-KeduTesting** [Buckinghamshire-Digital-Service/hr-service-desk-alpha#20]
currently mobile burger icon has aria-hidden=false when viewing desktop

![Menu_-_accessibility_-_on_desktop_the_mobile_burger_icon_should_have_aria-hidden=true](https://user-images.githubusercontent.com/2722522/64777135-7d22a300-d551-11e9-9fa5-cc5c905f9b17.png)

**Comment by enginehouse** [comment]
Fixed with this commit 13a55533 Debreaking the ie11 query string
